### PR TITLE
fix defaults for argument robust in tidy.felm

### PR DESCRIPTION
**Pull request by kuriwaki**
fixes my previous PR by giving tidy the same default that `lfe:::summary.felm` does. I show the confusion in: https://github.com/tidymodels/broom/issues/772#issuecomment-581097583

### CODE REVIEW

**Comment by simonpcouch** [comment]
Hi there! Apologies for the delay--looks great. Do you mind to merge upstream?

Also, linking to #772 and #781.

**Comment by kuriwaki** [comment]
I think I managed to merge upstream, and resolve the conflicts around lfe. @simonpcouch could you review?

**Comment by simonpcouch** [comment]
Thanks for your work! Will come back to this once we're able to merge the release candidate branch back into master.

File: R/lfe-tidiers.R [hunk 1]
```diff
@@ -59,7 +60,7 @@ tidy.felm <- function(x, conf.int = FALSE, conf.level = .95, fe = FALSE, robust
       select(response, dplyr::everything())
   } else {
     ret <- ret <- as_tidy_tibble(
```
Comment (grantmcdermott) [inline R/lfe-tidiers.R:62]: Why `ret <- ret <- `?

**Comment by simonpcouch** [comment]
Thanks for yalls patience and attention here!

There are a few small changes needed here and some whitespace and docs changes that need to be merged, and I don't want to ask any more of you, so I'm going to go ahead and push a commit that will incorporate changes from, and close, this PR.

**Comment by kuriwaki** [comment]
I just corrected a failing test, which is important for this PR, here.  https://github.com/kuriwaki/broom/commit/d84ab24b913c22299cf9364104a6a327eee19bd1

**Comment by simonpcouch** [comment]
That was one of the changes, yes! Will also drop in that new unit test from d84ab24. :-)

**Comment by grantmcdermott** [comment]
Hi folks. It's not related to this PR in particular, but I just saw one other thing that might be an issue for tidy.felm [here](https://github.com/kuriwaki/broom/blob/1780f152ddb59ca4e20f3802828563404ebeb89d/R/lfe-tidiers.R#L70).

Specifically, I don't think the "conf.int" argument is going to reflect the correct values when `robust = TRUE`, since the latter has to be passed to `summary.felm()` first...

**Comment by grantmcdermott** [comment]
@simonpcouch Sorry, I didn't see that you had closed this already. I'll open up a new issue about the conf.int problem.

**Comment by simonpcouch** [comment]
Ah, shoot. Not clear to me from the [`confint.felm` source](https://github.com/cran/lfe/blob/938de3428c8ea1afc5ea9c26f647a29992e89696/R/generics.R#L142-L156) how to specify the `robust` argument—thoughts?

Will keep an eye out for your issue. :-)

**Comment by grantmcdermott** [comment]
Ya, I think there's an easy fix. Will post the issue for providence and then submit a PR (famous last words!)

**Comment by akornelis** [comment]
First, huge thanks all for your work on this! I just updated broom to 0.7 (was previously using 0.5) and was surprised to discover that tidy no longer defaults to reporting clustered standard errors when taking a felm object with cluster. Is this still unresolved or is this the intended behavior?

`library(broom)
library(lfe)
library(dplyr)
package_version('lfe')
package_version('broom')

data = data.frame(var1 = runif(100),
                  var2 = runif(100),
                  var3 = rep(1:10, 10)) %>%
  mutate(y = 2*var1 + var2 + rnorm(100))

mod <- felm(data = data, y ~ var1 + var2 | 0 | 0 | var3)  

summary(mod)
#                   Estimate Cluster s.e. t value Pr(>|t|)    
# (Intercept)   0.3482       0.1878   1.854   0.0667 .  
# var1              1.7497       0.2392   7.316 7.38e-11 ***
# var2              1.1441       0.2689   4.255 4.83e-05 ***

tidy(mod)
# term        estimate std.error statistic    p.value
# <chr>          <dbl>     <dbl>     <dbl>      <dbl>
# 1 (Intercept)    0.348     0.275      1.27 0.208     
# 2 var1               1.75      0.345      5.07 0.00000191
# 3 var2               1.14      0.375      3.05 0.00294  `

**Comment by grantmcdermott** [comment]
@akornelis Try the dev version of broom (install from GitHub). That should fix everything.

**Comment by akornelis** [comment]
> @akornelis Try the dev version of broom (install from GitHub). That should fix everything.
> 
> PS. See the PR for more details. #920

Got it. Thanks Grant! Development version works great. Will these updates be in CRAN soon?

**Comment by simonpcouch** [comment]
Yes! Very soon, hopefully. :-)

**Pull request closed without merging**